apps/api/src/database/repositories/admin-repository.ts

```diff
@@ -309,73 +16,73 @@ async function countImpl(): Promise<number> {
  */
 
 export const create = createTimedRepositoryFunction(
-  createImpl,
+  repository.create.bind(repository),
   "AdminRepository",
   "create",
 );
```

Comment:
For consistency and simplicity and not having to update everywhere the repository functions are used, I left the timing wrapper definitions in place, just swapping out the old function being wrapped for the new one that lives on the repository instance.

It will be up to us to decide how/if to implement timing tracking like this in the comps app. When we do, it will probably not be like this and will use more integrated OpenTelemetry tooling.